stdlib/sys.mli

```diff
@@ -208,6 +208,12 @@ external runtime_parameters : unit -> string = "caml_runtime_parameters"
     as the contents of the [OCAMLRUNPARAM] environment variable.
     @since 4.03 *)
 
+external poll_runtime : unit -> unit = "%poll"
+(** Run any pending actions from the runtime, such as signal
+    handlers, finalizers, or memprof callbacks. Any allocation
```

Comment:
Poll points also handle GC requests, which could be worth pointing out explicitly.